web/blog/tenzir-platform-v1.7/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Tenzir Platform v1.7: Explorer Drag'n'Drop"
+slug: tenzir-platform-v1.7
+authors: [lava]
+date: 2025-01-06
+tags: [release, platform]
+comments: true
+---
+
+To kick off the new year, we're releasing [Tenzir Platform v1.7][github-release], featuring support for file drag and drop and a lot
+of stability improvements..
+
+![Tenzir Platform v1.7](tenzir-platform-v1.7.png)
+
+[github-release]: https://github.com/tenzir/platform/releases/tag/v1.7.0
+
+<!-- truncate -->
+
+## Explorer Drag'n'Drop
+
+It is now possible to drag and drop JSON files to the explorer window in order to create a pipeline reading the data from the dropped file.
```

Comment:
This is not restricted to JSON files. The feature supports any file through the new `from` operator in Tenzir Node v4.25.